src/constants.js

```diff
@@ -6,6 +6,9 @@
 
 import Browser from 'utils/Browser';
 
+/* ------------------------ API ---------------------------- */
+export const API_PAGE_LIMIT = 1000; // default limit for paginated api calls
```

Comment:
Can you change line 71ish export const METADATA_TEMPLATE_FETCH_LIMIT = ~~1000~~API_PAGE_LIMIT;